src/pedal/transparent.rs

```diff
@@ -0,0 +1,26 @@
+use crate::Pedal;
+
+#[derive(Debug)]
```

Comment:
`Debug` is one of two "to string" traits in Rust. (The other is `Display`, for more user-readable strings.) You use them like this:

```rust
println!("debug string: {:?}", some_struct);
println!("display string: {}", some_struct);
```

The compiler can derive the `Debug` implementation automatically, as long as all of the type of the fields of the type it's implementing it for also implement `Debug`. (It can also derive implementation of `Eq`, `Ord`, and a bunch of other stuff.) I tend to derive `Debug` on every type where it's allowed.